aspnet/tutorials/first-mvc-app/adding-model.rst

```diff
@@ -34,72 +31,64 @@ In the **Add Scaffold** dialog, tap **MVC Controller with views, using Entity Fr
+	- **Data context class:** *ApplicationDbContext(MvcMovie.Models)*
+	- **Views:**: Keep the default of each option checked
+	- **Controller name:** Keep the default *MoviesController*
+	- Tap **Add**
 
 .. image:: adding-model/_static/add_controller2.png
 
 The Visual Studio scaffolding engine creates the following:
 
-- A movies controller (MoviesController.cs)
-- Create, Delete, Details, Edit and Index Razor view files
-- Migrations classes
+- A movies controller (*Controllers/MoviesController.cs*)
+- Create, Delete, Details, Edit and Index Razor view files (*Views/Movies*)
+
+Visual Studio automatically created the `CRUD <https://en.wikipedia.org/wiki/Create,_read,_update_and_delete>`__ (create, read, update, and delete) action methods and views for you (the automatic creation of CRUD action methods and views is known as *scaffolding*). You'll soon have a fully functional web application that lets you create, list, edit, and delete movie entries.
+
+Run the app and click on the **Mvc Movie** link. You'll get the following error:
+
+.. code-block:: html
 
-  - The ``CreateIdentitySchema`` class creates the :doc:`membership database </security/authentication/identity>` tables. The Identity database stores user login information that is needed for authentication. We won't cover authentication in this tutorial, for that you can follow `Additional resources`_ at the end of this tutorial.
-  - The ``ApplicationDbContextModelSnapshot`` class creates the EF entities used to access the Identity database. We'll talk more about EF entities later in the tutorial.
+  A database operation failed while processing the request.
+  SqlException: Invalid object name 'Movie'. 
+  There are pending model changes for ApplicationDbContext
+  Scaffold a new migration for these changes and apply them to the database from the command line:
+  > dotnet ef migrations add [migration name] 
+  > dotnet ef database update 
 
-Visual Studio automatically created the CRUD (create, read, update, and delete) action methods and views for you (the automatic creation of CRUD action methods and views is known as *scaffolding*). You'll soon have a fully functional web application that lets you create, list, edit, and delete movie entries.
+That's a great error message, we'll follow those instructions to get the database ready for our Movie app.
 
 Use data migrations to create the database
 --------------------------------------------
 
 - Open a command prompt in the project directory (MvcMovie/src/MvcMovie). Follow these instructions for a quick way to open a folder in the project directory.
 
-  - Open a file in the root of the project (for this example, use *Startup.cs*.)
-  - Right click on *Startup.cs*  **> Open Containing Folder**.
+ 	- Open a file in the root of the project (for this example, use *Startup.cs*.)
+ 	- Right click on *Startup.cs*  **> Open Containing Folder**.	
 
-  .. image:: adding-model/_static/quick.png
+	.. image:: adding-model/_static/quick.png
```

Comment:
The code in this image is showing the old namespaces (i.e. Microsoft.AspNet.* instead of Microsoft.AspNetCore.*).